config/initializers/sentry.rb

```diff
@@ -1,5 +1,3 @@
-unless Rails.application.secrets.sentry_dsn.nil?
-  Raven.configure do |config|
-    config.dsn = Rails.application.secrets.sentry_dsn
-  end
+Sentry.init do |config|
+  config.dsn = Rails.application.secrets.sentry_dsn
```

Comment:
Need to init in order not to error in tests. It won't send events if sentry_dsn is empty.